packages/core/contracts/proxy-scripts/lsp-broker/LspUniswapV2Broker.sol

```diff
@@ -19,23 +21,55 @@ contract LspUniswapV2Broker {
 
     /**
      * @notice Mint long and short tokens and deposit them all into a UniV2 Pool.
+
```

Comment:
Can you add a @dev comment to this and the `mintAndSell` function about how the caller needs to have approved this contract so spend `amountCollateral` of the collateral token?

Reply:
sure.

Reply:
we should do that on `atomicMintSellOneSide` as well